test/test_bulk.js

```diff
@@ -243,24 +247,32 @@ describe('BULK:', function() {
       }));
   });
 
-  // TODO: unskip this.
-  // timeout is more than 100s
-  it.skip('addAgentContextOperationsBulk: should work with 10 agents with large number of operations', function() {
-    const agentIds = Array.apply(null, Array(10))
-      .map((x, i) => ({ id: `agent${i}_${RUN_ID}` }));
-    return client
-      .createAgentBulk(agentIds
-        .map(({ id }) => ({ id, configuration: CONFIGURATION_1 })))
-      .then(() => client.addAgentContextOperationsBulk(agentIds
+  it('addAgentContextOperationsBulk: should work with 10 agents with large number of operations', function() {
+    const agentIdsToTest = [
+      { id: agentIds[0] },
+      { id: agentIds[1] },
+      { id: agentIds[2] },
+      { id: agentIds[3] },
+      { id: agentIds[4] },
+      { id: agentIds[5] },
+      { id: agentIds[6] },
+      { id: agentIds[7] },
+      { id: agentIds[8] },
+      { id: agentIds[9] }
+    ];
+    return client.deleteAgentBulk(agentIdsToTest)
+      .then((deletions) => Promise.all(deletions))
+      .then(() => client.createAgentBulk(agentIdsToTest
```

Comment:
```suggestion
    return client.createAgentBulk(agentIdsToTest
```
The delete is already handle in the beforeEach